scipy/stats/_morestats.py

```diff
@@ -3022,24 +3022,34 @@ def mood(x, y, axis=0, alternative="two-sided"):
     return z, pval
 
 
-WilcoxonResult = _make_tuple_bunch('WilcoxonResult', ['statistic', 'pvalue'],
-                                   ['zstatistic'])
```

Comment:
We still need to use `_make_tuple_bunch`, at least for the 3-return case. Adding an element to the `namedtuple` breaks tests, as it should.